prog/dftb+/lib_common/blacsenv.F90

```diff
@@ -0,0 +1,155 @@
+!--------------------------------------------------------------------------------------------------!
+!  DFTB+: general package for performing fast atomistic simulations                                !
+!  Copyright (C) 2017  DFTB+ developers group                                                      !
+!                                                                                                  !
+!  See the LICENSE file for terms of usage and distribution.                                       !
+!--------------------------------------------------------------------------------------------------!
+
+#:include 'common.fypp'
+
+!> Contains BLACS environmental settings.
+module blacsenv
+  use mpienv
+  use message
+  use scalapackfx
+  use assert
+  implicit none
+  private
+
+  public :: TBlacsEnv, TBlacsEnv_init
+
+
+  !> Contains various BLACS related settings
+  type :: TBlacsEnv
+
+    !> Group grid for (nOrb, nOrb) shaped square matrices (nAtom) shaped vectors
+    !> Note: the grid is ensured to contain all processes in the group
```

Comment:
this grid always contains all processes in the group

Reply:
Sure, done.